pkg/operator/highcpuusagesnoalertcontroller/performanceprofile_test.go

```diff
@@ -0,0 +1,114 @@
+package highcpuusagesnoalertcontroller
+
+import (
+	"context"
+	"math/rand"
+	"strconv"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/fake"
+)
+
+func Test_performanceProfileControlPlaneCores(t *testing.T) {
+	type args struct {
+		ctx context.Context
+		cl  dynamic.Interface
+	}
+	tests := []struct {
+		name       string
+		args       args
+		coresFound int
```

Comment:
this is our expectation, please rename to `expectedCores` or `expectedCoreCount`